HHS/OPRE-OPS
# feat: ✨ round one of Change Requests on Approve page

## What changed

- first round of Approve Agreement page
- adds Review Changes Accordion
- adds Condensed Review Card
- removes some deprecated WorkFlow bits

## Issue

#2349 

## How to test

1. add 3 kinds of change requests (budget, status to PLANNED, and status to EXECUTING)
2. click on the 3 kinds of change requests from the review page
3. see approve page updates:
  - review changes accordion
  - condensed review card
4. 💰 profit

## Screenshots

_coming soon_

## Definition of Done Checklist
